src/test/scala/edu/ie3/simona/agent/em/EmAgentSpec.scala

```diff
@@ -276,8 +276,8 @@ class EmAgentSpec
       emAgentActivation ! Activation(0)
 
       // expect flex requests
```

Comment:
One could argue to change the requests in the comments here as well into activations.

Reply:
Thanks for looking out for this, I had missed it